.github/workflows/Build.yml

```diff
@@ -61,6 +61,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          python -m pip install --upgrade pip
+          pip install  pytest-cov
           pip install coveralls
```

Comment:
It should not be here 

Only under:
- name: 📃 Unittest and Integration tests with pytest